<launch>
    <arg name="num_iteration" default="10"/>
    <arg name="use_ros_lock" default="true"/>
    <arg name="duration_sleep" default="0.1"/>

    <node
        pkg="ros_lock"
        type="lock_manager_node.py"
        name="lock_manager"
        required="true"
        output="screen"
        />

    <node
        pkg="ros_lock"
        type="sample_client_roseus.l"
        name="sample_client_roseus"
        output="screen"
        >
        <rosparam subst_value="true">
            num_iteration: $(arg num_iteration)
            use_ros_lock: $(arg use_ros_lock)
            duration_sleep: $(arg duration_sleep)
        </rosparam>
    </node>

    <node
        pkg="ros_lock"
        type="sample_client_rospy.py"
        name="sample_client_rospy"
        output="screen"
        >
        <rosparam subst_value="true">
            num_iteration: $(arg num_iteration)
            use_ros_lock: $(arg use_ros_lock)
            duration_sleep: $(arg duration_sleep)
        </rosparam>
    </node>
</launch>
